stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -203,6 +203,8 @@
         <item name="streamUiMessageOptionsAvatarStyle">@style/StreamUi.MessageOptions.Avatar</item>
     </style>
 
+    <style name="StreamUiEmptyTheme"/>
```

Comment:
Why is it needed?

Reply:
We need a theme to create `ContextThemeWrapper` inside `createStreamThemeWrapper`, so we can change the UiMode. This is an empty theme for it doesn't override anything. Take a look in the description of the constructor of ContextThemeWrapper: 

> The specified theme will be applied on top of the base context's theme. Any attributes not explicitly defined in the theme identified by themeResId will retain their original values.

Reply:
Another comment deals with it in a better way